views/repository/204.jade

```diff
@@ -0,0 +1,17 @@
+extends ../layout
+
+block content
+  link(rel='stylesheet', href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css')
+  .errorDiv
+    .error-code.m-b-10.m-t-20
+      | 204
+      i.fa.fa-warning
+    h3.font-bold
+      | We could not find any build logs for&nbsp;
+      span(style="color: #db4545")=name
+    .error-desc
+      | It is possible that the repository is registered but there have been no commits afterwards!
+      div
+        a.login-detail-panel-button.btn(href='/')
```

Comment:
Good point about non-logged in users. Would it be possible to send logged in users to dashboard and non-logged in users to preview page? It's just that it was slightly irritating to having to click `dashboard`, then `settings` everytime.